acme/order.go

```diff
@@ -16,6 +17,9 @@ import (
 
 var defaultOrderExpiry = time.Hour * 24
 
+// Mutex for locking ordersByAccount index operations.
+var ordersByAccountMux = &sync.Mutex{}
```

Comment:
Not important but, `sync.Mutex` is usually used as a value. This should be equivalent.

```go
var orderByAccountMux sync.Mutex
```